paper/report_final.tex

```diff
@@ -0,0 +1,563 @@
+    \centering
+        \includegraphics[scale=0.4]{figures/Regression1/logistic_summary.png}
+    \caption{Results summary of the Logistic regression}
+\end{figure}
+
+\subsection{Linear Regression on BOLD data}
+
+The topic we are interested in exploring is whether loss aversion reflects 
+the engagement of distinct emotional processes when potential gains and 
+losses are considered. In the process, we want to explore the correlation 
+between neural and behavioral loss aversion in whole brain analysis. We also 
+want to try to identify the regions of brain that is more activated by this 
+loss aversion activity.
+
+Since we want to explore the correlation between neural and behavioral loss 
+aversion, the second step is to find out the neural loss aversion. In order 
+to find the neural loss aversion, we perform a linear regression on the BOLD 
+data against the parametric gain values and the parametric loss values, as 
+explained in our model section. (While implementing the linear regression for 
+the raw data, we also added linear and quadratic drift in our model. These 
+drift terms are modeling for gradual drifts across the time series.)
+
+We are especially interested in the beta coefficients of our parametric gain 
+and parametric loss regressors, which are the first two columns in our design 
+matrix. To find out the regions with significant positive or negative 
+correlation with increasing gain or loss levels, we calculate the t statistics 
+for each voxel. Plotting heat maps of the t statistics will show us regions 
+with significant parametric increase in fMRI signal to increasing potential 
+gains and regions with significant parametric decrease to increasing potential 
+losses.
+
+By looking at the heatmaps, we can get a general idea of how potential gains 
+and potential losses affect brain activation. We can also identify the areas 
+that have large coefficients; these are the areas that the brain activation is 
+highly connected to the potential gains and losses. We choose the preprocessed 
+data for subject 2 to plot heat maps since the preprocessed data is mapped 
+onto the standard brain. We plot slices 31 to 60 from the third dimension of 
+the brain (top to bottom). The red color is associated with positive t values 
+and blue color is associated with negative t values. We can see that for the 
+significant t values for the gain coefficients are mostly positive while the
+the significant t values for the loss coefficients are mostly negative.
+
+\begin{figure}[H]
+    \centering
+        \includegraphics[scale=0.4]{figures/Regression2/t_gain_standard_sub2.png}
+    \caption{t values of the gain coefficients for subject 2}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+        \includegraphics[scale=0.4]{figures/Regression2/t_loss_standard_sub2.png}
+    \caption{t values of the loss coefficients for subject 2}
+\end{figure}
+
+We can see that signigicant areas for the gain coefficients and those of the 
```

Comment:
can you replace this paragraph with this (fix some typoes): 

+We can see that significant areas for the gain coefficients and those of the 
+loss coefficients are mostly the same. This suggests the opposite of what 
+most people believe (that increasing potential losses should affect the areas 
+of the brain that mediate negative emotions in decision-making). Potential 
+losses were represented by decreasing activity in the same areas that are 
+sensitive to potential gains.